apps/prairielearn/src/ee/pages/instructorAiGenerateJobs/instructorAiGenerateJobs.html.ts

```diff
@@ -0,0 +1,71 @@
+import { formatDate } from '@prairielearn/formatter';
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { HeadContents } from '../../../components/HeadContents.html.js';
+import { JobStatus } from '../../../components/JobStatus.html.js';
+import { Job } from '../../../lib/db-types.js';
+
+export function AiGenerateJobReviewpage({
```

Comment:
Naming: `InstructorAiGenerateJobs`.

Reply:
Done in deabe2646069d6cce9d383460ea361768ab6c66c